src/odemis/gui/model/__init__.py

```diff
@@ -558,6 +570,10 @@ def __init__(self, main):
         # associated to the data displayed
         self.acq_fileinfo = VigilantAttribute(None) # a FileInfo
 
+        self.roa = model.TupleContinuous((0, 0, 1, 1),
```

Comment:
Why do you need to have a .roa in this class? Isn't it used only by the analysis tab? That tab doesn't provide a ROA.